pkgs/by-name/ut/util-linux/package.nix

```diff
@@ -15,32 +14,32 @@
   pam,
   systemdSupport ? lib.meta.availableOn stdenv.hostPlatform systemd,
   systemd,
+  sqlite,
   nlsSupport ? true,
   translateManpages ? true,
   po4a,
   installShellFiles,
   writeSupport ? stdenv.hostPlatform.isLinux,
   shadowSupport ? stdenv.hostPlatform.isLinux,
   gitUpdater,
+  fetchpatch,
 }:
 
 stdenv.mkDerivation rec {
   pname =
     "util-linux" + lib.optionalString (!nlsSupport && !ncursesSupport && !systemdSupport) "-minimal";
-  version = "2.39.4";
+  version = "2.40.4";
 
   src = fetchurl {
     url = "mirror://kernel/linux/utils/util-linux/v${lib.versions.majorMinor version}/util-linux-${version}.tar.xz";
-    hash = "sha256-bE+HI9r9QcOdk+y/FlCfyIwzzVvTJ3iArlodl6AU/Q4=";
+    hash = "sha256-XB2vczsE6YWa/cO9h8xIEYDuD4i1wJRrFv3skxl1+3k=";
   };
 
   patches = [
     ./rtcwake-search-PATH-for-shutdown.patch
-
     (fetchpatch {
-      name = "basename.patch";
-      url = "https://git.kernel.org/pub/scm/utils/util-linux/util-linux.git/patch/?id=77454e58d58f904cfdc02d3ca5bb65f1bd8739fc";
-      hash = "sha256-ELWC4bYN3rvn9XIN0XgCo55pXNfS2VpbZWuwzRLfO/0=";
+      url = "https://github.com/util-linux/util-linux/commit/e47c6f751a7ef87640c61316ada774e8e9cc6b07.patch";
```

Comment:
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Please vendor the patch, as it could disappear at any time following a rebase and GitHub garbage collection.

Reply:
makes sense, patch is now vendored